libs/saksbehandling-common/src/main/kotlin/behandling/Grunnlagsendringshendelse.kt

```diff
@@ -34,7 +35,10 @@ enum class Saksrolle {
             AVDOED -> PersonRolle.AVDOED
             GJENLEVENDE -> PersonRolle.GJENLEVENDE
             INNSENDER -> PersonRolle.INNSENDER
-            UKJENT -> throw Exception("Ukjent Saksrolle kan ikke castes til PersonRolle")
+            UKJENT -> {
+                sikkerlogger().warn("Ukjent saks")
```

Comment:
Skulle det vært logget noe fornuftig her? Trenger ikke sikkerlogg for å logge "ukjent saks" 😛

Reply:
Haha, sånn går det når ein prøver to løysingar samtidig